pkgs/development/python-modules/curated-transformers/default.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  curated-tokenizers,
+  huggingface-hub,
+  tokenizers,
+  torch,
+}:
+
+buildPythonPackage rec {
+  name = "curated-transformers";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "explosion";
+    repo = name;
+    rev = "v${version}";
+    hash = "sha256-QhJZnQIa9TilwdQCUlxnQCEc6Suj669cht6WHUAr/Gw=";
+  };
+
+  dependencies = [
+    curated-tokenizers
+    huggingface-hub
+    tokenizers
+    torch
+  ];
```

```suggestion
  dependencies = [
    torch
  ];
```
https://github.com/explosion/curated-transformers/blob/f2bca8f4e09d8f95ae1224de700a586cd6d15cce/setup.cfg#L16-L17

---

Upstream `setup.cfg` is wrong. I am one of the two developers of (spacy)?-curated-(tokenizers|transformers). I'll try to get it fixed for the next release.

---

I misremember, some of these can be removed, I'll fix the list.